doc/administrator/upgrades/index.md

```diff
@@ -11,6 +11,146 @@ description: "Upgrade Notes"
 
 The majority of IRIDA's upgrade notes can be seen at <https://github.com/phac-nml/irida/blob/master/UPGRADING.md>.  When there are more significant upgrades to the IRIDA system, database, deployment, etc. this page will go further in depth about how to properly back up your system, perform the upgrade, and recover from problems.
 
+# 21.01
+## Sample metadata audit record updates
+
+This upgrade contains some changes to how sample-metadata records are stored in IRIDA's database.  The intent of this update is to simplify the relationships between samples and metadata records to help with database performance and future developments.  It will also remove some redundancy in the database structure and remove any "dangling" metadata entries which have been removed from a sample, but not fully deleted from the database.
+
+While investigating how these updates will be applied to IRIDA databases, we noticed some irregularities in the audit records for some existing metadata.  Because of this we're recommending additional backup procedures be taken before completing this upgrade.  
+
+To skip straight to the backup procedure, [click here](#metadata-linking-script) For a full technical description of the issue, keep reading below.
+
+### Database structure
+{:.no_toc}
+
+To fully understand this problem, we'll first describe the structure of our metadata system and relevant database tables.  Given the following example of a basic line-list:
+
+|         | Organism   |
+|---------|------------|
+| sample1 | Salmonella |
```

Comment:
It's funny that you choose to use Organism for this example since it is technically still on the sample.  I think that update is something that needs to happen ASAP.